xdem/coreg/affine.py

```diff
@@ -677,16 +693,20 @@ def _to_matrix_func(self) -> NDArrayf:
         # If degree==0, it's just a bias correction
         empty_matrix = np.diag(np.ones(4, dtype=float))
 
-        empty_matrix[2, 3] += self._meta["coefficients"][0]
+        empty_matrix[2, 3] += self._meta["fit_params"][0]
 
         return empty_matrix
 
 
 class NuthKaab(AffineCoreg):
     """
-    Nuth and Kääb (2011) DEM coregistration: iterative registration of horizontal and vertical shift using slope/aspect.
+    Nuth and Kääb (2011) coregistration, https://doi.org/10.5194/tc-5-271-2011.
+
+    Estimate horizontal and vertical translations by iterative slope/aspect alignment.
```

Comment:
I would write "Estimate horizontal translations iteratively using a relationship between elevation difference, slope and aspect".
Also one thing that we need to discuss/clarify. The Nuth & Kaab (2011) papers suggest to correct the vertical misalignment using the parameters of the sinusoid fit, which we don't. We use a median of the elevation difference. We should maybe add this to the description. And do we want to make the reduction function optional or we assume median is the best? The latter would be easier and is probably fine. But this means that there is no need to follow the NuthKaab method with a VerticalShift.